examples/aws/storage-efs/README.md

```diff
@@ -0,0 +1,214 @@
+        --cidr $cidr_range
+    ```
+
+4. Create an Amazon EFS file system for your Amazon EKS cluster.
+```
+file_system_id=$(aws efs create-file-system \
+    --region $CLUSTER_REGION \
+    --performance-mode generalPurpose \
+    --query 'FileSystemId' \
+    --output text)
+```
+
+## 5.0 Create Mount Targets for your cluster
+1. [Optional] If you are re-using an existing EFS Volume, you will first have to delete any old mount targets. You can use the following commands for this - 
+```
+aws efs describe-mount-targets --file-system-id $file_system_id
+aws efs delete-mount-target --mount-target-id <each-id>
+```
+
+2. Determine the IP address of your cluster nodes.
+```
+kubectl get nodes
+```
+
+3. Determine the IDs of the subnets in your VPC and which Availability Zone the subnet is in.
+```
+aws ec2 describe-subnets \
+    --filters "Name=vpc-id,Values=$vpc_id" \
+    --query 'Subnets[*].{SubnetId: SubnetId,AvailabilityZone: AvailabilityZone,CidrBlock: CidrBlock}' \
+    --output table
+```
+
+4. Add mount targets for the subnets that your nodes are in. If are more than 1 nodes in the cluster, you'd run the command once for a subnet in each AZ that you had a node in, replacing subnet-EXAMPLEe2ba886490 with the appropriate subnet ID from the previous command
+```
+aws efs create-mount-target \
+    --file-system-id $file_system_id \
+    --security-groups $security_group_id \
+    --subnet-id subnet-EXAMPLEe2ba886490
+```
+
+
+## 6.0 Using the EFS Storage in Kubeflow
+
+### Sample 1 - Static Provisioning
+[Using this sample from official AWS Docs](https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/master/examples/kubernetes/multiple_pods) we have provided the required spec files in the sample subdirectory but you can create the PVC another way. 
+
+1. Use the `$file_system_id` you recorded before or use the following command to get the efs filesystem id - 
+```
+aws efs describe-file-systems --query "FileSystems[*].FileSystemId" --output text
+```
+
+2. Now edit the sample/pv.yaml and edit the `volumeHandle` to point to your EFS filesystem
+
+3. Now create the required persistentvolume, persistentvolumeclaim and storageclass resources as -
+```
+kubectl apply -f examples/aws/storage-efs/sample/pv.yaml
+kubectl apply -f examples/aws/storage-efs/sample/pvc.yaml
+kubectl apply -f examples/aws/storage-efs/sample/sc.yaml
+```
+
+
+## 7.0 Connecting to Central dashboard
+Logon to http://localhost:8080 using default credentials.
+
+## 8.0 Test your Setup
+Check the `Volumes` tab in Kubeflow and you should be able to see your PVC is available for use within Kubeflow as follows - 
+
+### 8.1 Additional Permissions 
+You might need to specify some additional directory permissions on your worker node before you can use these as mount points. The set-permission-job.yaml is an example of how you could set these permissions to be able to use the efs as your workspace in your kubeflow notebook. Edit the spec file as needed for your usecase
```

Comment:
Can you also what the current  supplied spec file is doing ?

Reply:
This particular spec file is picked up from the official kubeflow docs - https://www.kubeflow.org/docs/distributions/aws/customizing-aws/storage/

I can add a similar description, yes.

Reply:
> > I also noticed that perviously the entire driver deploy section was getting copied here.
> > Is there a way we can avoid this ?
> 
> Not Sure I follow this comment, could you elaborate a little.

I am saying this change has entire efs driver code....was wondering if there is a way to dynamically pull it. Same thing was being done in previous release. So see if there is any better way to import it. 

It saves us by not duplicating the EFS driver code in this repo.

Reply:
Definitely doable by directly applying from the upstream repos but in that case our README will look pretty similar to their set of instructions. 
We want to delete the `distributions/aws/aws-efs-csi-driver` directory entirely in that case, correct?

Link - https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/release-1.3#installation

Reply:
see my reply to next comment for more details.